src/rgw/rgw_common.cc

```diff
@@ -2411,6 +2442,19 @@ bool RGWBucketInfo::empty_sync_policy() const
   return sync_policy->empty();
 }
 
+std::string RGWBucketInfo::dump_flags() const {
+  return flag_summary(
+    {
+      { BUCKET_SUSPENDED, "suspended" },
+      { BUCKET_VERSIONED, "versioned" },
+      { BUCKET_VERSIONS_SUSPENDED, "versions_suspended" },
+      { BUCKET_DATASYNC_DISABLED, "datasync_disabled" },
+      { BUCKET_MFA_ENABLED, "mfa_enabled" },
+      { BUCKET_OBJ_LOCK_ENABLED, "obj_lock_enabled" },
+    },
```

Comment:
please add a comment to `enum RGWBucketFlags` that reminds developers to update this when adding new flags